python/tests/integration/arcticdb/version_store/test_basic_version_store.py

```diff
@@ -781,10 +799,14 @@ def test_empty_ndarr(basic_store):
     basic_store.write(sym, ndarr)
     assert_array_equal(basic_store.read(sym).data, ndarr)
 
+# The following restrictions should be checked in the cpp layer's name_validation
+MAX_SYMBOL_SIZE=255
+UNSUPPORTED_S3_CHARS={'*', '<', '>'}
```

Comment:
Why isn't "\0" in this set like it used to be? Are we checking it by some other mechanism now?

Reply:
The \0 is now checked by limiting the range of characters to [32, 126]. I added some explicit checks in the test_unhandled_chars*